apps/backoffice-v2/src/pages/Entities/components/CaseCreation/components/CaseCreationForm/hooks/useCaseCreationForm/utils/prepare-context.ts

```diff
@@ -0,0 +1,16 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { generateEntityId } from '@/pages/Entities/components/CaseCreation/components/CaseCreationForm/hooks/useCaseCreationForm/utils/generate-entity-id';
+import { AnyObject } from '@ballerine/ui';
+
+export const prepareContext = (formData: AnyObject): TWorkflowById['context'] => {
```

Comment:
I think the function's name can be clearer. We already have many variants like "merge", "create", "build", and "compose" just to name a few. What does "prepare" do?